inst/pages/introductory_workflow.qmd

```diff
@@ -146,7 +146,7 @@ manipulation using [**the dplyr package**](https://dplyr.tidyverse.org/).
 ### Subsetting
 
 In some cases, you may only need to use part of your original
-*TreeSummarizedExperiment*.
+`*TreeSummarizedExperiment*`.
```

Comment:
This should be \`TreeSummarizedExperiment\` without additional styling. Check line 107 also

Reply:
Sure.